Models/SSBTaxAssessment/GraphQLTypes/TaxAssessmentDatasetType.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Backend.Models.Base.JsonStat;
+using Backend.Models.SSBTaxAssessment.POCO;
+using Backend.utils;
+using GraphQL;
+using GraphQL.Types;
+
+namespace Backend.Models.SSBTaxAssessment.GraphQLTypes
+{
+    public class
+        TaxAssessmentDatasetType : JsonStatDatasetType<TaxAssessment.TaxAssessmentDimension, TaxAssessmentDimensionType>
+    {
+        public TaxAssessmentDatasetType() : base(true)
+        {
+            Field<ListGraphType<LabeledValueTaxType>>("value", "The values for the selected municipalities and years.",
+                null,
+                context =>
+                {
+                    var municipalities = context.Parent.GetArgument<List<string>>("municipalities");
+                    var years = context.Parent.GetArgument<List<string>>("years");
+                    var skipSize = context.Source.Dimension.Size[3];
+                    var municipalityEntrySize = context.Source.Dimension.Size[0];
+                    return (from municipality in municipalities
+                        let municipalityYears = context.Source.Value
+                            .Skip(NorwayTools.MunicipalityCodeToIndexTaxes[municipality] * 294)
+                            .Take(municipalityEntrySize)
+                            .ToList()
+                        select new LabeledValueTax
+                        {
+                            Municipality = NorwayTools.MunicipalityCodeToMunicipalityNameTaxes[municipality],
+                            TaxesForYear = new Collection<TaxesForAGivenYear>(years.Select(year =>
+                                    new TaxesForAGivenYear
+                                    {
+                                        Year = year,
+                                        Brutto = municipalityYears[NorwayTools.YearToIndexTaxes[year] * skipSize],
```

Comment:
Can we use English variable names?

Reply:
We can, they are based of the codes in "contents" over at https://data.ssb.no/api/v0/dataset/49607?lang=en, which are the same for what language you have selected, so just thought it would be easier if they had the same name as they do in the data source.

Reply:
My only concern is that it mixes languages in the code which is a slight no-no as I'm sure @blauks can tell you more about.

Reply:
I see that, my concern is that if we rename them again, we will have to come up with our own descriptive names. Which means that trying to understand what the different properties in the class are, would make even less sense. I think it would be clearer if we have a description at the top of the class (as we do now) saying why the class names are derived from the SSB-codes.

Reply:
Sounds reasonable!